ups_modul/inout/inout.py

```diff
@@ -171,11 +170,12 @@ def loop():
 # weird stuff
 if __name__ == "__main__":
     setup_inout()
-    connected = False
-    while connected == False:
+    connected = "OFF"
+    while connected == "OFF":
         try:
             check_new_users()
-            connected = True
+            connected = "ON"
         except:
-            sleep(100)
+            time.sleep(100)
+        print(connected)
```

Comment:
Do we need this print function?

Reply:
just for checking whether the script is on or not, handy when debuging